NuCivic/dkan
# Remove tab groups and add link member list 907

Removed tabs on node groups and added link on sidebar to see member list.

Steps to test:
  * Access to a node group.
  * Add a new user.
  * You shouldn't see any tab on body and see a link Member list on the sidebar.

https://github.com/NuCivic/internal/issues/907
